pmda/density.py

```diff
@@ -0,0 +1,144 @@
+import numpy as np
+import MDAnalysis as mda
+import time
+from MDAnalysis.lib.util import fixedwidth_bins
+from MDAnalysis.analysis.density import Density
+from .parallel import ParallelAnalysisBase
+
+class DensityAnalysis(ParallelAnalysisBase):
+    """Parallel density analysis.
+
+    Parameters
+    ----------
+    universe : MDAnalysis.Universe
+            :class:`MDAnalysis.Universe` object with a trajectory
+    atomselection : str (optional)
+            selection string (MDAnalysis syntax) for the species to be analyzed
+            ["name OH2"]
+    delta : float (optional)
+            bin size for the density grid in Angstroem (same in x,y,z) [1.0]
+    start : int (optional)
+    stop : int (optional)
+    step : int (optional)
+            Slice the trajectory as "trajectory[start:stop:step]"; default
+            is to read the whole trajectory.
+    metadata : dict. (optional)
+            `dict` of additional data to be saved with the object; the meta data
+            are passed through as they are.
+    padding : float (optional)
+            increase histogram dimensions by padding (on top of initial box
+            size) in Angstroem. Padding is ignored when setting a user defined
+            grid. [2.0]
+    update_selection : bool (optional)
+            Should the selection of atoms be updated for every step? ["False"]
+            - "True": atom selection is updated for each frame, can be slow
+            - "False": atoms are only selected at the beginning
+    parameters : dict (optional)
+            `dict` with some special parameters for :class:`Density` (see docs)
+    gridcenter : numpy ndarray, float32 (optional)
+            3 element numpy array detailing the x, y and z coordinates of the
+            center of a user defined grid box in Angstroem ["None"]
+    xdim : float (optional)
+            User defined x dimension box edge in ångström; ignored if
+            gridcenter is "None"
+    ydim : float (optional)
+            User defined y dimension box edge in ångström; ignored if
+            gridcenter is "None"
+    zdim : float (optional)
+            User defined z dimension box edge in ångström; ignored if
+            gridcenter is "None"
+    """
+    def __init__(self, atomgroup, delta=1.0, atomselection="name OH2",
```

Comment:
Make the default `atomselection=None` – this should only be needed in cases when `updating=True`